models/user.go

```diff
@@ -1,3 +1,17 @@
-package models 
+package models
 
-type UserId int64
\ No newline at end of file
+import "strings"
+
+type UserId int64
+
+type EmailAddress struct {
```

Comment:
Seems weird for this to be in `user.go`. Any reason not to create `email.go` or something?

Reply:
I consider UserId and Email address to be part of the User model.

Reply:
Hmm, yeah i might be generalizing too quickly again. As of now, what you're saying is correct, so yeah let's leave it here